modules/javafx.graphics/src/main/java/javafx/concurrent/Service.java

```diff
@@ -99,8 +99,8 @@
  * <p>
  *     Because a Service is intended to simplify declarative use cases, subclasses
  *     should expose as properties the input parameters to the work to be done.
- *     For example, suppose I wanted to write a Service which read the first line
- *     from any URL and returned it as a String. Such a Service might be defined,
+ *     For example, suppose I wanted to write a Service that reads the first line
```

Comment:
As long as you are changing this sentence, can you also change `I` to `you`? The unintended use of first person here is a bit jarring.

Reply:
I rephrased the sentence.